docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/onvif-user-authentication.md

```diff
@@ -0,0 +1,129 @@
+# Onvif User Authentication
+
+According to the Onvif user authentication flow, the device service shall:
+* Implement WS-Usernametoken according to WS-security as covered by the core specification.
+* Implement HTTP Digest as covered by the core specification.
+
+The spec can refer to https://www.onvif.org/specs/core/ONVIF-Core-Specification.pdf
+
+![onvif-user-authentication](../images/onvif-user-authentication.jpg)
+
+## Usage
+The user need to define the **AuthMode** and **SecretName**, and device service will send SOAP action with **WS-Usernametoken** or **Digest header**.
+
+For example:
+```yaml
+[[DeviceList]]
+Name = "test-camera"
+ProfileName = "camera"
+Description = "HIKVISION camera"
+  [DeviceList.Protocols]
+    [DeviceList.Protocols.Onvif]
+    Address = "192.168.12.123"
+    Port = 80
+    AuthMode = "usernametoken"
+    SecretName = "credentials001"
+```
+
+The AuthMode can be:
+* digest
+* usernametoken
+* both
+* none
+
+SecretName should contain:
+* username
+* password
+
+For development purpose, we can define the secrets in the configuration.yaml
+```yaml
+Writable:
+...
+  InsecureSecrets:
+    Camera001:
+      SecretName: credentials001
+      SecretData:
+        username: administrator
+        password: Password1
+    Camera002:
+      SecretName: credentials002
+      SecretData:
+        username: administrator
+        password: Password1
+```
```

Comment:
this stuff is no longer correct and should probably be removed

Reply:
Should I just remove this file?

Reply:
i mean the bottom stuff still has valid info about how Onvif itself works, so it is fine to leave it and just remove the top portion